src/gc/gcinterface.d

```diff
@@ -37,6 +37,10 @@ struct Range
 
 interface GC
 {
+    /// Provided for implementation's convenience, not intended for public usage
+    protected alias CollectionStartHook = void delegate() nothrow @nogc;
+    /// Ditto
+    protected alias CollectionEndHook = void delegate(size_t, size_t) nothrow @nogc;
```

Comment:
`@__future`? Although I don't think it's working for aliases.

Reply:
I don't think that's necessary for `alias` actually. If someone defines something with the same name, it will just shadow this.